app/posts/register-of-training-providers/2025-09-01-releasing-the-first-slice-of-the-register-of-training-providers.md

```diff
@@ -0,0 +1,64 @@
+
+This release was not a full service, but a minimum set of features that demonstrated how key parts of the register would work together.
+
+Releasing this early version enables us to test our assumptions, gather feedback, and build confidence in the direction of the service.
+
+## What we released
+
+The first slice included:
+
+- DfE Sign-in integration - so users could securely log in using an existing DfE account
+- support user management - administrators could add, edit or delete support users
+- a ‘Your account’ page - to view and check account details
+- adding and managing providers - users could add a provider with basic details (legal name, operating name, UKPRN, URN and provider code), and then edit, delete, archive or restore them, with validation built in
+- provider search and filters - users could search, filter and paginate a list of providers
+- static content pages - including the start page, accessibility statement, cookie policy and privacy policy
+
+This set of features focused on the foundations: who can access the system, how we manage users, and how provider records are created, changed and found.
+
+## Why we released early
+
+We wanted to test the value of the register with real users and stakeholders as early as possible. Releasing a small, working slice gives us several advantages, including:
+
+- validating the core user journey, especially around searching for and managing providers
+- getting early feedback from policy and operational teams, helping us shape priorities and clarify assumptions
+- reducing delivery risk by proving we can integrate with DfE Sign-in, manage user accounts, and enforce data validation before building more complex features
+- helping stakeholders gain confidence since they can see and interact with the service, rather than just reviewing slide decks or documents
+
+## Benefits of this approach
+
+Releasing early and often is a core principle of agile delivery. It allows us to:
+
+- iterate with purpose - getting feedback from real usage helps us prioritise improvements and identify gaps
+- spot issues early - we can uncover edge cases and usability challenges that we would not have found through design reviews alone
+- build trust - stakeholders can see progress and understand how their input shaped the service
+
+## Challenges and trade-offs
+
+This approach also comes with some challenges, including:
+
+- incomplete features can confuse users - we needed to clearly explain the purpose of the release and manage expectations around what the service could and could not do
+- additional effort is needed to test and support partial functionality - we needed to monitor usage and triage feedback even for this limited slice
+- stakeholders may expect progress to be faster - early visibility can raise expectations, so we had to communicate clearly about what would come next
+
+## What we hope to learn
```

Comment:
Maybe we should also call out principles or practice we are following as these will stand against test of time.

“delivery principles” supported by “delivery practices”

With `Release early, release often` as the principle and `Minimum Viable Product` as the practice.